protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -122,8 +142,8 @@ class FieldInfo<T> {
     return () => defaultOrMaker;
   }
 
-  /// Returns `true` if this represents a dummy field standing in for a field
-  /// that has been removed by a program transformation.
+  /// `true` if this represents a dummy field standing in for a field that has
```

Comment:
Another boolean getter which can use "Whether" (also to avoid starting with a lower-case letter). 
Try to avoid the unqualified "this".

Maybe:
```dart
/// Whether this field is a dummy field.
///
/// A dummy field represents a field that has been removed by a program transformation.
```
(Anything else we can say about dummy fields, like "don't use them for anything"?)

This is a private field, so the documentation doesn't need to be up to public-API standards.